Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Source.jinja

```diff
@@ -914,10 +914,16 @@ enum class NetworkProperties
             return;
         }
 
+{%     if (ReplicateFrom == 'Authority' and ReplicateTo == 'Server') or (ReplicateTo == 'Client' and Property.attrib['IsPredictable'] | booleanTrue == false) %}
```

Comment:
What does the `| booleanTrue == false` mean in this context? Just seeing it in the PR, it looks either meaningless (always false?) or horrifying (sometimes true?) ...

Reply:
It pipes the IsPredictable property into a helper function that deduces whether it's a true value or not.. This just abstracts away all the numerous ways people might type a true value into a helper.. 'T' 't' 'True' 'true' 'TRUE' '1', etc..

Reply:
Thanks for the explanation! I'm not familiar with our Jinja setup, so I didn't realize "booleanTrue" is a helper method (defined in AzAutoGen.py from the looks of it). The helper method name would've been easier for me to understand as a casual reader if it was called StringToBoolean or something, but that ship has long sailed. Anyways, I'll understand it going forward, thanks!